src/citrus/core.cljs

```diff
@@ -22,19 +22,21 @@
     config              - a map of
       state             - app state atom
       controllers       - a hash of state controllers
+      default-handler   - a function to handle incoming events (see docs)
```

Comment:
Should we link to the document here?

Reply:
changed this to say `(see doc/default-handler.md)`